pkg/util/defaults/defaults.go

```diff
@@ -42,4 +42,7 @@ const (
 
 	// ImageName --
 	ImageName = "docker.io/apache/camel-k"
+
+	// CamelVersion --
+	CamelVersion = "3.6.0"
```

Comment:
This is a readonly file, which is generated from Makefile options. We've removed the need to specify the Camel version, because a runtime version implies only one Camel version and one Quarkus version.

Instead of hardcoding it here, you can load the CamelCatalog for the current runtime version (it's embedded in the `/deploy` virtual file system) and retrieve it from the metadata section (https://github.com/apache/camel-k/blob/c79dc9b40ddaa82c38fca405aa484ce60ce5983d/deploy/camel-catalog-1.5.0-quarkus.yaml#L34).

Reply:
Done, thanks for the feedback.